etc/helm/pachyderm/values.yaml

```diff
@@ -239,6 +239,51 @@ loki-stack:
     config:
       lokiAddress: http://{{ .Release.Name }}-loki:3100/loki/api/v1/push
       snippets:
+        scrapeConfigs: |
+          - job_name: kubernetes-pods
+            pipeline_stages:
+              {{- toYaml .Values.config.snippets.pipelineStages | nindent 4 }}
```

Comment:
I had no idea you could use template variables in the values file, but I guess it works for lokiAddress above!

Reply:
Yeah I think it only works because its being passed to the subchart and the subchart is doing something with it there, helm is kind of terrifying honestly.